diff --git a/node_modules/pkce-challenge/dist/index.node.js b/node_modules/pkce-challenge/dist/index.node.js
index bb108cb..0eebfc2 100644
--- a/node_modules/pkce-challenge/dist/index.node.js
+++ b/node_modules/pkce-challenge/dist/index.node.js
@@ -1,8 +1,11 @@
 let crypto;
 crypto =
     globalThis.crypto?.webcrypto ?? // Node.js 16 REPL has globalThis.crypto as node:crypto
-        globalThis.crypto ?? // Node.js 18+ 
+    globalThis.crypto ?? // Node.js 18+
+    // (await import("node:crypto")).webcrypto; // Node.js 16 non-REPL
+    (async () => {
         (await import("node:crypto")).webcrypto; // Node.js 16 non-REPL
+    })();
 /**
  * Creates an array of length `size` of random bytes
  * @param size
